bodhi/tests/server/services/test_releases.py

```diff
@@ -327,3 +339,49 @@ def test_query_releases_html_two_releases_same_state(self):
         self.assertEquals(res.content_type, 'text/html')
         self.assertIn('Fedora 22', res)
         self.assertIn('Fedora 42', res)
+
+    def test_get_critpath_packages(self):
+        res = self.app.get('/releases/F22/critpath-packages')
+        packages = res.json_body['packages']
+
+        self.assertEqual(len(packages), 1)
+        self.assertEqual(packages[0]['name'], u'bodhi')
+
+    def test_add_critpath_packages(self):
+        self.app.put('/releases/F22/critpath-packages',
+                     {'packages': 'python,json,nodejs,git'})
+
+        release = self.db.query(Release).filter(Release.name == 'F22').one()
+        pkg_list = [pkg.name for pkg in release.critpath_pkgs]
+
+        self.assertEqual(set(pkg_list), {'bodhi', 'python', 'json', 'nodejs', 'git'})
```

Comment:
Rather than creating a list and then a set, you can use a set comprehension on line 355 above to get it as a set in the first place. Same for other places that do this below.